File: seleniumTestCssPath.cs
using System;
using OpenQA.Selenium;
using OpenQA.Selenium.Chrome;
using System.Threading;
class EntryPoint
{
    static void Main()
    {
        string url = "https://testing.todorvachev.com/css-path/";
        string cssPath = "#search-2 > form > label > input";

        IWebDriver driver = new ChromeDriver();

        driver.Navigate().GoToUrl(url);

        IWebElement element = driver.FindElement(By.CssSelector(cssPath));

        if (element.Displayed)
        {
            Console.WriteLine("Yes - I can see the element!");
        } else
        {
            Console.WriteLine("No, the element is not there.");
        }

        Thread.Sleep(7000);

        driver.Quit();
    }
}

File: Java/SeleniumTestCssPath.java
package com.selenium;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;

public class Main {

    public static void main(String[] args) throws InterruptedException {
	// write your code here
        String url = "https://testing.todorvachev.com/css-path/";
        String cssPath = "#search-2 > form > label > input";

        WebDriver driver = new ChromeDriver();

        driver.get(url);

        WebElement element = driver.findElement((By.cssSelector(cssPath)));

        if (element.isDisplayed())
        {
            System.out.println("Yes - I can see the element!");
        } else
        {
            System.out.println("No, the element is not there.");
        }

        Thread.sleep(7000);

        driver.quit();
    }
}
